src/clients/chunker.rs

```diff
@@ -60,13 +68,24 @@ impl ChunkerClient {
         model_id: &str,
         request: Pin<Box<dyn Stream<Item = BidiStreamingTokenizationTaskRequest> + Send + 'static>>,
     ) -> Result<ReceiverStream<TokenizationStreamResult>, Error> {
+        let (tx, rx) = mpsc::channel(128);
+        // Handle "default" separately first
+        if model_id == DEFAULT_MODEL_ID {
+            info!("Using default whole doc chunker");
+            let whole_response_stream = bidi_streaming_tokenize_whole_doc(request).await;
+            tokio::spawn(async move {
+                if let Ok(message) = whole_response_stream {
+                    let _ = tx.send(message).await;
+                }
+            });
+            return Ok(ReceiverStream::new(rx));
```

Comment:
I was thinking since this is essentially static if we can get away with spawning tokio task and passing along the response via channel here? Or is that not straightforward because of the type mismatch ?

Reply:
Or have we added this since its streaming input as well so we have to wait and thus is blocking?